br/pkg/utils/backoff.go

```diff
@@ -343,6 +347,7 @@ func NewDiskCheckBackoffStrategy() BackoffStrategy {
 	return NewBackoffStrategy(
 		WithRemainingAttempts(resetTSRetryTime),
 		WithDelayTime(resetTSWaitInterval),
+		WithMaxDelayTime(resetTSMaxWaitInterval),
```

Comment:
can those change break the br behavior?

Reply:
sounds good, I think that's a good point. will open another one